jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -1,37 +1,100 @@
 <% require "digest"%>
+<%# Ruby Variables to make the template more readable -%>
+<%
+# Stats Binding Variables {{{
+stat = p("ha_proxy.stats_bind").split(':')
+stat_prefix = stat[0] + ":";
+stat_port = stat[1].to_i;
+# }}}
+# Accept Proxy {{{
+accept_proxy = ""
+if p("ha_proxy.accept_proxy") == true
+    accept_proxy = "accept-proxy"
+end
+# }}}
+# Global SSL Flags {{{
+ssl_flags = "no-sslv3"
+if p("ha_proxy.disable_tls_10")
+	ssl_flags = "#{ssl_flags} no-tlsv10"
+end
+if p("ha_proxy.disable_tls_11")
+	ssl_flags = "#{ssl_flags} no-tlsv11"
+end
+if p("ha_proxy.disable_tls_tickets")
+	ssl_flags = "#{ssl_flags} no-tls-tickets"
+end
+# }}}
+# TLS Bind Options {{{
+mutual_tls_enabled = p("ha_proxy.client_cert")
+ssl_enabled = false
+crt_config = ""
+if_p("ha_proxy.ssl_pem") do
+    ssl_enabled = true
+    crt_config="crt /var/vcap/jobs/haproxy/config/ssl"
+    if p("ha_proxy.client_ca_file") || p("ha_proxy.client_revocation_list")
```

Comment:
The expected behaviour of this condition should be:
Set `mutual_tls_enabled ` to true if either `ha_proxy.client_ca_file` or `ha_proxy.client_revocation_list` is configured.

But if `ha_proxy.client_ca_file` is not configured, you get the following error:
```
      - Error filling in template 'haproxy.config.erb' (line 34: Can't find property '["ha_proxy.client_ca_file"]')
```

The following code runs without errors (but it's not really elegant)
```
    if_p("ha_proxy.client_ca_file") do
        mutual_tls_enabled = true
    end
    if_p("ha_proxy. client_revocation_list") do
        mutual_tls_enabled = true
    end
```